src/ActionModal/components/ModalWithdrawRewards.vue

```diff
@@ -62,10 +62,23 @@ export default {
         type: transaction.WITHDRAW
       }
     },
+    isMultiDenomNetwork() {
+      return this.rewards.length > 1 &&
+        this.rewards[0].denom !== this.rewards[1].denom
```

Comment:
How about:

this.rewards.find(({ denom }) => denom !== this.network.stakingDenom)

Reply:
Yes, also a possibility. I thought about it but saw, in this case, both would work the same (since the else is doing pretty much the same). But I will change it